features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1195,6 +1195,15 @@
         {% if oauth.extensions.scope_metadata_service is defined %}
         <ScopeMetadataService>{{oauth.extensions.scope_metadata_service}}</ScopeMetadataService>
         {% endif %}
+
+        {% if oauth.restricted_query_parameters is defined %}
```

Comment:
since we have the default.json values, we don't need this is defined check right?

Reply:
Fixed with d35455710cb1658652659409542dec0673a99cdb